frontend/js/src/settings/Settings.tsx

```diff
@@ -111,33 +136,55 @@ export default function Settings() {
         </div>
 
         <div className="mb-15 donation-flairs-settings">
-          <div className="form-group">
+          <form className="form-group" onSubmit={submitFlairPreferences}>
             <h3>Flair Settings</h3>
             <p>
-              Choose which flair you want your username to show to let other
-              users know you donated. ListenBrainz.
+              Unlocked by <Link to="/donate/">donating</Link>. Some flairs are
```

Comment:
Can we also add a modal here explaining how do we calculate the time duration for which the flairs remain?

Reply:
Yes, I think this is crucial and currently missing, I had the same thought.

Reply:
Here's what I have, would love to hear @Aerozol's feedback as well:
![image](https://github.com/user-attachments/assets/04abcbf8-b9c9-4a58-a9c1-46db9e2440d5)

Reply:
I thought this was important too, glad to see you're already onto it!

My usual text changes, take or leave as you please:
"Every $5 donated unlocks flair for 1 month. Larger donations will unlock multiple months of flair. Multiple donations will stack."
(I assume this is all correct now, after lucifer changed the donation calculation system?)

Two additional points (sorry!)

1. We will have to mention that the flair isn't forever on the donations page as well - we can also add the same modal/modal text.
2. Since it's getting a bit confusing we should probably show how many months of flair are left/have been built up :S This seems like the right place for it. e.g. "You have x days of flair remaining". (display nothing if they have not donated)

Reply:
Well I messed up and pushed commits to the feature branch directly instead of opening  PR...

But here it is, using the existing nag-check endpoint to get the remaining days of unlock, or warn the user otherwise:
![image](https://github.com/user-attachments/assets/45ee4ad7-2ba2-452e-9c54-58345ca8cb06)

![image](https://github.com/user-attachments/assets/278dd24d-ce25-4d2d-aaa8-db0d30751af4)

Tooltip is the same message as just above:
![image](https://github.com/user-attachments/assets/aa3ec5b3-0e18-4f13-b8bf-460ea9a81284)